### DISCUSSION TITLE
Plot the subject

### DISCUSSION THREAD

**Discussion by vijaypshah** [TorchIO-project/torchio#441] [General]
Hi,

When I display the subject using .plot() it seems to be using neurological format... is it  . Is there a way to specify that data is displayed in "radiology" format so  that the "patient left"  is on the right of image  ?

Thanks 
Best Regards,
Vijay

**Comment by fepegar** [comment]
Hi, @vijaypshah. Good point. I have modified the behavior of `plot` to show images in radiological convention by default. It's available in `v0.18.24`.

<img width="1280" alt="Screen Shot 2021-02-03 at 09 37 02" src="https://user-images.githubusercontent.com/12688084/106728165-1b5e4f80-6604-11eb-9309-519603f0242c.png">

**Comment by vijaypshah** [comment]
Thank you @fepegar ! It works with new version! 

Best Regards,
Vijay
